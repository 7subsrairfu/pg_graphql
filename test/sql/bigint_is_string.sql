begin;

    create table account(
        id bigserial primary key,
        parent_id int references account(id)
    );

    insert into account(id, parent_id)
    values (1,1);

    select graphql.resolve($$
    {
      accountCollection {
        edges {
          cursor
          node {
            id
            parent {
              id
            }
          }
        }
      }
    }
    $$);

rollback;
