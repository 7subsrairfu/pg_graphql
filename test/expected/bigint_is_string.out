begin;
    create table account(
        id bigserial primary key,
        parent_id int references account(id)
    );
    insert into account(id, parent_id)
    values (1,1);
    select graphql.resolve($$
    {
      accountCollection {
        edges {
          cursor
          node {
            id
            parent {
              id
            }
          }
        }
      }
    }
    $$);
                                                                 resolve                                                                  
------------------------------------------------------------------------------------------------------------------------------------------
 {"data": {"accountCollection": {"edges": [{"node": {"id": 1, "parent": {"id": "1"}}, "cursor": "WyJhY2NvdW50IiwgMV0="}]}}, "errors": []}
(1 row)

rollback;
